feg/gateway/services/s8_proxy/servicers/ie_conversions.go

```diff
@@ -170,8 +170,28 @@ func getUserLocationIndication(mcc, mnc string, uli *protos.UserLocationInformat
 func getRatType(ratType protos.RATType) *ie.IE {
 	var rType uint8
 	switch ratType {
+	case protos.RATType_RESERVED:
```

Comment:
Doesn't protobuf provide a function to convert between enums and its value? Or is the assignment slightly different from the actual value?